### ISSUE TITLE
Spike - accessibility scanner replacement

### ISSUE THREAD

**Issue by TKDickson** [department-of-veterans-affairs/va-mobile-app#3846]
## Description 
As a mobile app QA, I want to test for contrast and touch target size (as well as other accessibility objectives that aren't solely part of the Accessibility Scanner), so that all kinds of users think our app works well.

As part of this story we want to:
- Figure out if there's a reasonable tool/set of tools we can use to replace the functionality we're missing with accessibility scanner (an upgrade of theirs really doesn't play well with our app, see #3772)

Previously, color contrast and touch target were (essentially) solely tested via accessbility scanner, and it was a backup method for checking other portions of our accessibility testing. 

## Acceptance Criteria
  
- Come up with a recommendation for how we test color contrast & touch target 
- Communicate that to the mobile QA team

**Comment by TKDickson** [comment]
Closing as not doing. With an accessibility specialist on the team, and moving towards accessibility annotations with all designs, this is no longer needed.
